projects/cashmere/src/lib/picklist2/pick-option.component.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    AfterViewChecked,
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    Input,
+    OnChanges,
+    OnDestroy,
+    SimpleChanges
+} from '@angular/core';
+import { Subject } from 'rxjs';
+
+/** Component used to add options to a picklist in declarative way. `<hc-pick-option>` */
+@Component({
+    selector: 'hc-pick-option',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    template: `<ng-content></ng-content>`
+})
+export class PickOptionComponent implements OnChanges, AfterViewChecked, OnDestroy {
+    /** Value for this option. */
+    @Input() value: any;
+    /** If true, the option cannot be selected. */
+    @Input() set disabled(value: boolean) { this._disabled = this._isDisabled(value); }
+    get disabled() { return this._disabled; }
+
+    readonly _stateChange$ = new Subject<{ value: any, disabled: boolean, label?: string }>();
```

Comment:
It's only used a few times here, but my personal preference would be to create a separate type/interface for:
```
{
value: any,
disabled: boolean,
label?: string
}

Reply:
Good ask, made this change.